VCFv4.1.tex

```diff
@@ -1358,11 +1357,14 @@ \subsubsection{Encoding the INFO fields}
 HM3;AC=3;AN=6;AA=C
 Let's assume that the header dictionary elements for HM3, AC, AN, and AA are at 80, 81, 82, and 83 respectively.  All of these can be encoded by 1-element INT8 values (0x11), with associated hex values of 0x50, 0x51, 0x52, and 0x53 respectively.
 
-First is HM3.  The entry begins with the key: 0x11 0x50.  Next we have a Flag value to indicate the field is present, represented as a 1 element INT8 value of 1.  Altogether we have:
+First is HM3.  The entry begins with the key: 0x11 0x50.
+The HM3 key has no value and therefore is represented as the special case of a missing value without an explicit type provided.
```

Comment:
I think I would find this confusing if I came on it without the current context.  I might rework it to include the word `flag`.  
```suggestion
The HM3 key is a flag with no associated value and therefore is represented as the special case of a missing value without an explicit type provided.
```